apiary.apib

```diff
@@ -518,6 +518,28 @@ Get a list of all the definitions of custom fields.
 
     + Attributes (object)
         + page (Page, optional)
+        + filter (object, optional)
+            + context (enum[string], optional)
+                + Members
+                    + contact
+                    + company
+                    + form
```

Comment:
B : this is not an existing type as far as the API is concerned
this is deprecated and we don't want it in the new API